source/jormungandr/jormungandr/realtime_schedule/tests/sytral_test.py

```diff
@@ -350,6 +412,28 @@ def next_passage_for_route_point_test(mock_good_response):
         assert passages[1].is_real_time
 
 
+def next_passage_for_route_point_with_direction_type_test(mock_direction_type_is_forward_response):
+    """
+    test the whole next_passage_for_route_point with direction_type parameter
+    mock the http call to return a good response, we should get some next_passages
+    """
+    sytral = Sytral(id='tata', service_url='http://bob.com/')
+
+    mock_requests = MockRequests(
+        {'http://bob.com/?stop_id=42&direction_type=forward': (mock_direction_type_is_forward_response, 200)}
+    )
+
+    route_point = MockRoutePoint(line_code='05', stop_id='42', direction_type='forward')
+
+    with mock.patch('requests.get', mock_requests.get):
+        passages = sytral.next_passage_for_route_point(route_point)
+
+        assert len(passages) == 1
+
+        assert passages[0].datetime == datetime.datetime(2016, 4, 11, 12, 37, 15, tzinfo=pytz.UTC)
+        assert passages[0].is_real_time
+
+
```

Comment:
Could you add a test with &direction_type=backward and another test with direction_type=all or both ?

Reply:
It is easy to add others direction_type test but I think that is not relevant. Why ?
Because in the test, no matter the value of the parameter. 
We write in the mock the same value as the URL. 
There is not intelligence behind.